budget_control/models/budget_period.py

```diff
@@ -519,7 +522,7 @@ def get_budget_info(self, analytic_ids):
         )
         return budget_info
 
-    def _set_budget_info_amount(self, source, domain, kwargs):
+    def _set_budget_info_amount(self, source, domain, kwargs, is_commit=False):
```

Comment:
Adding an option `is_commit` to make this function a little smarter. So that I don't always have to repeat `_set_budget_info_amount` in every module.

Code can be shorter too.